lib/url.cpp

```diff
@@ -203,6 +203,7 @@ void escape_url_readable(char *in, char* out) {
 // or prepend it
 //   - Remove double slashes in the rest
 //   - Add a trailing slash if necessary
+//   - Convert all alphabet characters to lower case
```

Comment:
Should it explicitly be indicated that only ASCII characters are relevant in this case.
Do we need to do anything specific, regarding Unicode URLs (I'm not familiar with how those are handled by BOINC)?